alpaca_trade_api/common.py

```diff
@@ -41,6 +41,26 @@ def __new__(cls, value):
         return str.__new__(cls, value)
 
 
+class FLOAT(str):
+    """
+    api allows passing floats or float as strings.
+    let's make sure that param passed is one of the two, so we don't pass
+    invalid strings all the way to the servers.
+    """
+    def __new__(cls, value):
+        if not value:
+            raise ValueError('Unexpected empty string')
+        if isinstance(value, float) or isinstance(value, int):
+            pass  # we're good
+        elif isinstance(value, str):
+            value = value.strip()  # make sure no spaces
+            if isinstance(float(value), float):
+                pass  # we're good
```

Comment:
if not? or if you are expecting exception what's the point of the `if`?

Reply:
I guess it's good for readability. reader know we've thought about this scenario